tests/cypress/integration/pdv_hoc_site_spec.js

```diff
@@ -0,0 +1,487 @@
+describe('Test H.O.C.', () => {
+
+  it('Coveo search', () => {
+    // out of scope
+    // not testable in local environments
+  })
+
+  it('Google Analytics', () => {
+    // out of scope
+    // not testable in local environments
+  })
+
+  it('Qualtrics feedback', () => {
+    // out of scope
+    // not testable in local environments
+  })
+
+  it('Classic/V1 release notes (en-US)', () => {
+    cy.visit('en-us/Content/Topics/ReleaseNotes/okta-relnotes.htm')
+
+  })
+
+  it('OIE release notes (en-US)', () => {
+    cy.visit('oie/en-us/Content/Topics/ReleaseNotes/oie-relnotes.htm')
+
+  })
+
+  it('Product documentation landing page (en-US)', () => {
+    cy.visit('en-us/Content/index.htm')
+    // Logo image
+    cy.get('img[title="Okta Support"]')
+      .should('be.visible')
+      .and(($img) => {
+        expect($img[0].naturalWidth).to.equal(382)
+      })
+    // Copyright
+    cy.get('p.copyright')
+      .contains('©')
+    // Menu dropdown
+    cy.get('div.navbar > div.dropdown')
+      .should(($div) => {
+        expect($div).to.have.length(5)
+      })
+    cy.get('div.dropdown-content')
+      .eq(0)
+      .should('be.hidden')
+      .invoke('show')
+    cy.get('div.dropdown-content > div > a')
+      .should(($a) => {
```

Comment:
we need to iterate through all the links right? I think we need to use `.each()` and then check that each of them have a value for `href` attribute
I am no expert on cypress, but I think it should be something like
```
cy.get("div.dropdown-content > div > a")
  .each($a => {
    cy.wrap($a).should("have.attr", "href")
  }) 
```

Reply:
Added